usr/share/rear/output/default/94_grub2_rescue.sh

```diff
@@ -110,6 +110,9 @@ else
     grub-mkconfig -o $grub_conf
 fi
 
+#Finding UUID of filesystem containing /boot 
+grub_boot_uuid=$(df /boot | awk 'END {print $1}' | xargs blkid -s UUID -o value)
```

Comment:
@gdha Ok, I'll do that.
Last question, what is the difference between `StopIfError `and `BugIfError` ?